lib/rspec/support/version_checker.rb

```diff
@@ -1,53 +0,0 @@
-module RSpec
-  module Support
-    LibraryVersionTooLowError = Class.new(StandardError)
-
-    # @private
-    class VersionChecker
```

Comment:
I was wrong. It's still used [here](https://github.com/rspec/rspec-rails/blob/078b29fefb129b7942ba512cc0173b0c6d20c582/lib/rspec/rails/vendor/capybara.rb#L12-L13). The `vendor` directory was ignored by my `ag`...